unity-renderer/Assets/Tutorial/Scripts/TutorialController.cs

```diff
@@ -177,11 +177,8 @@ internal void SetupTutorial(string fromDeepLink, string enableNewTutorialCamera,
             NotificationsController.disableWelcomeNotification = true;
 
             WebInterface.SetDelightedSurveyEnabled(false);
-
-            if (!CommonScriptableObjects.rendererState.Get())
-                CommonScriptableObjects.rendererState.OnChange += OnRenderingStateChanged;
-            else
-                OnRenderingStateChanged(true, false);
+            
+            CommonScriptableObjects.rendererState.OnChange += OnRenderingStateChanged;
```

Comment:
It is needed to keep the `if (!CommonScriptableObjects.rendererState.Get())` condition because there are some times when we need to start directly the tutorial while we are already in the world (with the `rendererState` as `true`). For instance clicking in the [TUTORIAL] button from the settings window:

![image](https://user-images.githubusercontent.com/64659061/176660100-02a3144d-9b25-4c7b-a6ae-b0828d321ab9.png)